helpers/go/build-go.sh

```diff
@@ -12,12 +12,11 @@ done
 
 GOVER="${GOVER:?}"
 
-export GOROOT_FINAL="/usr/libexec/go-${GOVER}"
 cd "${HOME}/sdk-go/src"
 ./all.bash
 
 # Install the Go standard library and toolchain.
 cd "${HOME}/sdk-go"
-PATH="${HOME}/sdk-go/bin:${PATH}" GO111MODULE="auto"
+export PATH="${HOME}/sdk-go/bin:${PATH}" GO111MODULE="auto"
```

Comment:
Why export these? They'll only be available to child processes of the shell, and the only one we care about is the `go` process that runs on the next line.

Reply:
Hm yea you're right, I misinterpreted a shellcheck that `GO111MODULE` is unused, I'll revert and add a `# shellcheck disable=SC2034`